client/my-sites/marketing/index.js

```diff
@@ -69,7 +69,6 @@ export default function () {
 		'/marketing/sharing-buttons/:domain',
 		siteSelection,
 		navigation,
-		jetpackModuleActive( 'sharedaddy' ),
```

Comment:
Seems like we can remove the `jetpackModuleActive` import as it's no longer in use.

Reply:
https://github.com/Automattic/wp-calypso/pull/53694